### PR TITLE
[7.x] As per the rest of the file state the type of cookie: encrypted

### PR DESCRIPTION
**Pull request by u01jmg3**
- As per the rest of the file state the type of cookie: encrypted 🔒
- Correct closing tags
- Formatting

### CODE REVIEW

File: dusk.md [hunk 1]
```diff
@@ -1533,9 +1533,9 @@ As shown above, a "date picker" is an example of a component that might exist th
         {
             return [
                 '@date-field' => 'input.datepicker-input',
-                '@year-list' => 'div > div.datepicker-years',
+                '@year-list'  => 'div > div.datepicker-years',
                 '@month-list' => 'div > div.datepicker-months',
-                '@day-list' => 'div > div.datepicker-days',
+                '@day-list'   => 'div > div.datepicker-days',
```
Comment (driesvints) [inline dusk.md:1538]: Please revert all indenting here.

Comment (u01jmg3) [reply dusk.md]: Apologies - I thought your standard was to align. Fixed now

Comment (driesvints) [reply dusk.md]: Did you see this anywhere else? Otherwise we can update those places as well (in a different pr/commit).

Comment (u01jmg3) [reply dusk.md]: Checked through all the docs and instances occur in [`database.md`](https://github.com/laravel/docs/blob/7.x/database.md), [`passport.md`](https://github.com/laravel/docs/blob/7.x/passport.md) and [`validation.md`](https://github.com/laravel/docs/blob/7.x/validation.md).

Also, on my travels I found instances where trailing commas were missing from arrays and 10 files with trailing whitespace that hasn't been trimmed and may cause a conflict in a future merge.

PRs ready, just let me know which corrections you'll accept!

Comment (driesvints) [reply dusk.md]: @u01jmg3 feel free to send in a different pr for those.

Comment (u01jmg3) [reply dusk.md]: Done

**Pull request merged**